docs/DafnyRef/Attributes.md

```diff
@@ -344,136 +326,217 @@ recursion was explicitly requested.
+                
+#### 22.2.2.5. `{:vcs_max_keep_going_splits N}` {#sec-vcs_max_keep_going_splits}
+
+Per-method version of the command-line option `/vcsMaxKeepGoingSplits`.
+If set to more than 1, activates the _keep going mode_ where, after the first round of splitting,
+[assertion batches](#sec-assertion-batches) that timed out are split into <n> [assertion batches](#sec-assertion-batches) and retried
+until we succeed proving them, or there is only one
+single assertion that it timeouts (in which
+case error is reported for that assertion).
+Defaults to 1.
+If [`{:vcs_split_on_every_assert}`](#sec-vcs_split_on_every_assert) is set, then this parameter is useless.
+
+#### 22.2.2.6. `{:vcs_split_on_every_assert}` {#sec-vcs_split_on_every_assert}
+Per-method version of the command-line option `/vcsSplitOnEveryAssert`.
+
+In the first verification round, this option will split the original [assertion batch](#sec-assertion-batches)
+into one assertion batch per assertion.
+This is mostly helpful for debugging which assertion is taking the most time to prove, e.g. to profile them.
+
+#### 22.2.2.7. `{:selective_checking true}`
+Turn all assertions into assumptions except for the ones reachable from after the
+assertions marked with the attribute `{:start_checking_here}`.
+Thus, `assume {:start_checking_here} something;` becomes an inverse
+of `assume false;`: the first one disables all verification before
+it, and the second one disables all verification after.
+
+#### 22.2.2.8. `{:priority N}`
+Assign a positive priority 'N' to an implementation to control the order
+in which implementations are verified (default: N = 1).
+
+#### 22.2.2.9. `{:id <string>}`
+Assign a unique ID to an implementation to be used for verification
+result caching (default: "<impl. name>:0").
+
+#### 22.2.2.10. `{:timeLimit N}`
+Set the time limit for a given implementation.
+
+### 22.2.3. Verification attributes on functions
+
+#### 22.2.3.1. `{:builtin "spec"}`
+#### 22.2.3.2. `{:bvbuiltin "spec"}`
+Rewrite the function to built-in prover function symbol 'fn'.
+
+#### 22.2.3.3. `{:inline}`
+`{:inline}` or `{:inline true}` expands a function according to its definition before going to the prover.
+
+#### 22.2.3.4. `{:never_pattern true}`
+Terms starting with this function symbol will never be
+automatically selected as patterns. It does not prevent them
+from being used inside the triggers, and does not affect explicit
+trigger annotations. Internally it works by adding `{:nopats ...}`
+annotations to quantifiers.
+
+#### 22.2.3.5. `{:identity}`
+If the function annotated with `{:identity}` or `{:identity true}` has 1 argument and the use of it has type `X->X` for
+some `X`, then the abstract interpreter will treat the function as an
+identity function.  Note, the abstract interpreter trusts the
+attribute--it does not try to verify that the function really is an
+identity function on the abstract domain.
```

Comment:
This Boogie attributes don't make sense in a Dafny program. I suggest these not be mentioned in the Dafny reference manual.